pages/common/travis.md

```diff
@@ -0,0 +1,24 @@
+# travis
+
+> Command line client to interface with a Travis CI service.
+> More information: <https://github.com/travis-ci/travis.rb>.
+
+- Display the client version:
+
+`travis version`
+
+- Authenticate the CLI client against the server, using an authentication token:
+
+`travis login`
+
+- List repositories the user has certain permissions on:
+
+`travis repos`
+
+- Encrypt values for the .travis.yml:
+
+`travis encrypt {{token}}`
+
+- Generate a .travis.yml and enable the project:
```

Comment:
Perhaps this will read better?
```suggestion
- Generate a `.travis.yml` file and enable the project:
```

Also, what precisely is meant by "enabling the project"? Enable it to do what?

Reply:
See https://github.com/travis-ci/travis.rb#enable. Enabling a project activates it on Travis CI.

Reply:
Ah I see. Sounds it has technical meaning then! The above suggestion should be fine then.